radixdlt-java-common/src/main/java/com/radixdlt/crypto/ECKeyPair.java

```diff
@@ -70,7 +70,7 @@ public static ECKeyPair generateNew() {
 			ECPrivateKeyParameters privParams = (ECPrivateKeyParameters) keypair.getPrivate();
 			ECPublicKeyParameters pubParams = (ECPublicKeyParameters) keypair.getPublic();
 
-			final ECPublicKey publicKey = ECPublicKey.fromBytes(pubParams.getQ().getEncoded(true));
+			final ECPublicKey publicKey = new ECPublicKey(pubParams.getQ());
```

Comment:
I think it's better to have factory method for this rather than call constructor directly.